andela/rivendell-ah-client
# #160056368: setup test coverage

#### What does this PR do?
- installs jest and enzyme
- added jest.config to the app
- updates CircleCI config to run tests
- adds project to Code-climate
- ensures that build is passing on Circleci
- separates script for test and coverage
- adds repo token environmental variable to CircleCI
-adds badges to the README

#### Description of Task to be completed?
- acquire test coverage badge from code-climate
- configure test environment
- configure CircleCI to run test

#### How should this be manually tested?
- run `git pull` from console
- run `npm run test` to see test result
- run `npm run coverage` to see coverage results
- go to circleCi to view  branch build via https://circleci.com/gh/andela/rivendell-ah-client/127

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
- 160056368 
- 160056367 
- 160056371

#### Screenshots (if appropriate)
<img width="1162" alt="screen shot 2018-09-06 at 1 23 49 am" src="https://user-images.githubusercontent.com/38565349/45148432-2b696f00-b1bf-11e8-8003-c937137ac99d.png">

#### Questions:
- None
